doc/readme.html

```diff
@@ -336,10 +336,19 @@ <h4>Selecting optional library features</h4>
 
         <li>
             <p>
-                <i>64bit indices</i>: By default, deal.II use unsigned int (32bit) indices for degrees of freedom, using the <code>types::global_dof_index</code> type. This limits the number of unknowns to approximately four billions. If larger problem
-                must be solved, pass the
+                <i>64bit indices</i>: By default, deal.II uses
+                unsigned int (32bit) indices for degrees of freedom,
+                using the <code>types::global_dof_index</code> type.
+                (The same is true for a few other index types such as the
+                global number of cells.) This limits the number of
+                cells or unknowns to approximately four billions. If
+                larger problem must be solved, pass the
                 <code>-DDEAL_II_WITH_64BIT_INDICES=ON</code> argument to
-                <code>cmake</code>. You will not be able to solve problems of this size on a single machine, but only when using clusters of computers and linear algebra packages PETSc or Trilinos. To use this option with PETSc, PETSc must be compiled
+                <code>cmake</code>. You will not be able to solve
+                problems of this size on a single machine, but only
```

Comment:
I don't think that is true any more. If you have a system with 512 GB of RAM, which is a bit more than the regular workstation/cluster but not that unusual either, you can solve systems of 6-8 billion dofs with explicit time integrators like the one from step-67 (#8319). I suggest to insert `typically`.

Reply:
Good point. I was going to add "unless you have exceptional amounts of memory and patience", but I'm sure that even that will become moot over time :-)